src/Contract/ScriptLookups.purs

```diff
@@ -78,14 +80,13 @@ import Types.TypedValidator
 -- | a separate call. In particular, this should be called in conjuction with
 -- | `balanceAndSignTx`.
 mkUnbalancedTx
-  :: forall (r :: Row Type) (a :: Type) (b :: Type)
-   . DatumType a b
-  => RedeemerType a b
-  => FromData b
-  => ToData b
-  => ScriptLookups.ScriptLookups a
-  -> TxConstraints b b
-  -> Contract r
+  :: forall (e :: Row Type) (v :: Type) (d :: Type) (r :: Type)
```

Comment:
Can we have longer names here and everywhere:

`e` -> `r`
`v` -> `validator`
`d` -> `datum`
`r` -> `redeemer`

Reply:
Everywhere, where `v`, `d` and `r` are?

Reply:
...or only where 4 typevars are?

Reply:
Preferably everywhere in scope of this PR